### ISSUE TITLE
Delegation List vs. Delegation ID

### ISSUE THREAD

**Issue by psychomb** [Emurgo/yoroi-frontend#1886]
The current UI of Yoroi extension presents two panels for delegation. 

The first presents a list of Stake Pool (pulled from ADAPools.org) while the second panel offers to delegate to a pool via its pool ID. 

**Issue**: 
Why not having a single delegation panel ? Since the delegation by list also offers the option to search for a pool via pool ID. This might be a bit confusing for first time users.

**Comment by nicarq** [comment]
@psychomb that's a great point, we will internally discuss. We left both of them, because one if native while the other depends on an external provider.

**Comment by psychomb** [comment]
I have just realized this issue now that you have pushed new icons in the UI. It eats quite some space and in the french translation for example (that I help maintaining), this does not look clean at all. Just trying to save some width space here :)

**Comment by nicarq** [comment]
@psychomb I will check with @Marta-Ryzhok what we can do for that as well

**Comment by SebastienGllmt** [comment]
Yes, we have a ticket internally to merge these together into a single tab to save space because we will need the extra space for stuff like a voting tab

**Comment by jarstorm** [comment]
Hello, and thank you for your contribution.
We're currently reviewing long-standing issues to keep the tracker manageable. As this issue hasn't seen activity in some time, we're closing it.
If the problem persists or you'd like to follow up, feel free to comment and we'll reopen it.
